tests/integration/test_rocksdb_options/test.py

```diff
@@ -149,6 +149,31 @@ def test_invalid_column_family_options(start_cluster):
     node.restart_clickhouse()
 
 
+def test_invalid_block_based_table_options(start_cluster):
+    node.exec_in_container(
+        [
+            "bash",
+            "-c",
+            "sed -i 's/block_size/no_such_block_based_table_options/g' /etc/clickhouse-server/config.d/rocksdb.xml",
```

Comment:
Why not just have a several configs in this tests and start several nodes with different setups?

Reply:
I don't see how running multiple instances better then sed + restart, more instances - likely will eat more resources, and there are problems with stability of integration tests.